src/RandomForestRegression.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+var RandomForestBase = require('./RandomForestBase');
+var Stats = require('ml-stat');
+
+var selectionMethods = {
+    mean: Stats.array.mean,
+    median: Stats.array.median
+};
+
+/**
+ * @class RandomForestRegression
+ * @augments RandomForestBase
+ */
+class RandomForestRegression extends RandomForestBase {
+
+    /**
+     * Create a new base random forest for a classifier or regression model.
+     * @constructor
+     * @param {object} options
+     * @param {number} [options.maxFeatures] - the number of features used on each estimator.
+     *        * if is an integer it selects maxFeatures elements over the sample features.
+     *        * if is a float between (0, 1), it takes the percentage of features.
+     * @param {boolean} [options.replacement] - use replacement over the sample features.
+     * @param {number} [options.seed] - seed for feature and samples selection, must be a 32-bit integer.
+     * @param {number} [options.nEstimators] - number of estimator to use.
+     * @param {object} [options.treeOptions] - options for the tree classifier, see [ml-cart]{@link https://mljs.github.io/decision-tree-cart/}
+     * @param {string} [options.selectionMethod] - the way to calculate the prediction from estimators, "mean" and "median" are supported.
+     * @param {object} model - for load purposes.
+     */
+    constructor(options, model) {
+        if (options === true) {
+            super(true, model.baseModel);
+        } else {
+            if (options === undefined) options = {};
```

Comment:
for the options you could use [`Object.assign`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) or [destructuring assignment](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), usually it's shorter